app/Http/Controllers/TestResultController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use App\Test;
+use App\Result;
+use Illuminate\Http\Request;
+use App\Http\Resources\TestResult;
+use App\Http\Resources\TesteeTestResult;
+use App\Http\Resources\Result as ResultDetails;
+
+class TestResultController extends Controller
+{
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  $id
```

Comment:
If you don't have a type hint in the method declaration than you should specify **mixed** as a type.